server/utilis/validation/signupValidation.js

```diff
@@ -0,0 +1,9 @@
+const joi = require('joi');
+
+const signupValidation = joi.object({
+  firstName: joi.string().required(),
+  lastName: joi.string().required(),
```

Comment:
please add minimum values to the first name, and make sure that the first name and last are alphabet or numbers,
```suggestion
  firstName: joi.string().alphanum().min(3).required(),
  lastName: joi.string().alphanum().min(3).required(),
```